README.adoc

```diff
@@ -559,6 +603,28 @@ added into the tree of Arquillian Report output file in every case.
 
 |===
 
+==== Arquillian Recorder Integration with Arquillian Redmine Governor Extension
+
+How to get Redmine issues,force value in arquillian reports?
+
+In order to see all issues & force value in arquillian reports, you have to put arquillian-recorder-reporter dependency in your Arquillian maven test.
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.arquillian.extension</groupId>
+    <artifactId>arquillian-recorder-reporter-impl</artifactId>
+    <version>${version.arquillian.recorder}</version>
+    <scope>test</scope>
+</dependency>
+----
+
+Configure arquillian-recorder-reporter as your requirement. Follow steps from here - https://github.com/arquillian/arquillian-recorder/wiki/Reporter
+
+You will get Redmine's all issues & force values i.e. true or false information located in target/arquillian-report.xml report.
```

Comment:
force value has two options which you can set using `@Redmine(Value="1", force=true)`